### DISCUSSION TITLE
Classic Max combo ccalculate broken when replay not loaded

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#20106] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/34775378/188254136-d0cb940e-f00e-41f2-b70d-541207ea7aaa.png)
![image](https://user-images.githubusercontent.com/34775378/188254143-14f8e74c-06b1-46d3-8870-07039afb82e8.png)
![image](https://user-images.githubusercontent.com/34775378/188254146-85e024f4-649d-47d1-92d6-6cfd595c1f80.png)

after replay finish
![image](https://user-images.githubusercontent.com/34775378/188254170-4155b421-6078-466c-bdc0-2ea404699ce7.png)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/20090

**Reply by cdwcgt** [reply]
is this problem also occurs with newly imported scores?
This happens to my newly imported score too, not just the old score

**Reply by frenzibyte** [reply]
That covers newly imported legacy scores, yes.
